package/yast2-network.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-network
-Version:        4.4.49
+Version:        4.4.50
```

Comment:
version doesn't match to the changelog one

Reply:
Ups :P